src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +56,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString(){
+        return "Phone:" + this.value;
```

Comment:
`Phone:` is a magic value that you should put it as a static final variable at the top of your program so that modifications can be done easily.